Images_And_Graphics_style_v1.css

```diff
@@ -345,18 +389,18 @@ ul li:hover ul {
 .Logo2, .Video1, .Image2 .Image6 .Image7 {
 	margin-top: 210px;
 }
-
 .Image3, .Image4 {
 	margin-top: -900px;
 	text-align: center;
 }
 
-.Image5 {
-	margin-left: 200px;
-	margin-top: -390px;
-	margin-bottom: 400px;
+.TawaLogo {

```

Comment:
I got rid of the class image5 and changed it to Tawa logo so I know what each image I added in my css is for.